veggieseasons/lib/screens/search.dart

```diff
@@ -79,12 +85,16 @@ class _SearchScreenState extends State<SearchScreen> {
       builder: (context) {
         return SafeArea(
           bottom: false,
-          child: Column(
+          child: Stack(
             children: [
-              _createSearchBox(),
-              Expanded(
-                child: _buildSearchResults(model.searchVeggies(terms)),
+              Column(
```

Comment:
Is this Column still necessary, now that it only has a single child? Could you instead use Positioned instead of Column/Expanded together?

Reply:
Respected Sir,
It's good to know that you responded so quick on my pr. Actually I was a bit unsure as to why the team was using Expanded in the previous code as similar thing can also be achieved without using the same. I had to use Column as well because documentation says Expanded should be a direct child of a Column or Row. Nevertheless I will me more than happy to make this change so that it can be merged with the current project.